refactor footer: remove resources section, update background, improve…

r1g023 opened the pull request:
### Updates

- Removed the resources box and dark background as per Val's recommendation. Also adjusted the TechStack section to span 66% width.
- Logos are now stacked on top of each other when resized.
- Additional padding has been added around the TechStack component for better spacing.

### QA

- Consider adding further enhancements to logo sizing for improved responsiveness.

### Attachments

- **Before Update**  
![beforeFooterUpdate](https://github.com/user-attachments/assets/b2fa8971-f8cf-4589-8f97-ff512095b0b9)

- **After Update**  
![afterFooterUpdate](https://github.com/user-attachments/assets/1d583104-da59-484e-bed7-8003986c8c48)

ss-deep reviewed (approved):
Great work, @r1g023! I just updated the README with my name, maybe you can uncomment me now 😄. Sorry I haven’t been keeping up, but I wanted to check: are we planning to use MongoDB?

r1g023 commented:
> Great work, @r1g023! I just updated the README with my name, maybe you can uncomment me now 😄. Sorry I haven’t been keeping up, but I wanted to check: are we planning to use MongoDB?

@ss-deep 
 - I’ve updated the Footer to include your name as well, as part of PR #51: [[PR Link](https://github.com/chingu-voyages/v52-tier3-team-40/pull/51)](https://github.com/chingu-voyages/v52-tier3-team-40/pull/51).  

Also I believe @Valeriusdev mentioned that we’ll be using MongoDB. But he can clarify further if needed
